web/stripe-template/api/routes/webhook/POST-subscription.ts

```diff
@@ -2,26 +2,26 @@ import { RouteContext } from "gadget-server";
 import { getStripeWebhookEvent } from "../../stripe";
 import { objKeyConvert, destructure } from "../../utils";
 
-/**
- * Route handler for POST webhook/subscription
- *
- * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
- *
- */
+export type StripeWebhookEvent = {
+  type: string;
+  data: {
+    object: any;
+  };
+};
+
 export default async function route({
```

Comment:
Same comment as the other route - we should do something like
```
const route: RouteHandler<{
  Body: StripeWebhookEvent;
  };
}> = async ({
  request,
  reply,
  api,
  logger,
  connections,
  currentAppUrl,
}) => {
    let event = request.body;
```

and export the route

Reply:
Fixed